### PR TITLE
LPD-23969 Add workflow Activities tab to the entity's detail screen

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-23969

### CODE REVIEW

**Comment by brookedalton** [comment]
https://github.com/liferay-publications/liferay-portal/assets/46908843/0148c1ff-a151-4e8b-84c7-c080bbd385d6

**Comment by brookedalton** [comment]
ci:test:sf

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js [hunk 1]
```diff
@@ -3,94 +3,145 @@
...
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							{workflowData.comments}
+						</ClayTable.Cell>
+					</ClayTable.Row>
+				</ClayTable.Body>
+			</ClayTable>
+
+			<ClayPanel
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js:100]: Is it possible to have nested tables? I think the spacing will look nicer if we have the ClayPanel in the ClayTable

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js [hunk 2]
```diff
@@ -3,94 +3,145 @@
...
+				collapsable
+				displayTitle={
+					<ClayPanel.Title className="publications-key-td table-cell-expand-small">
+						<b> {Liferay.Language.get('activities')} </b>
+					</ClayPanel.Title>
+				}
+				displayType="primary"
+				showCollapseIcon={true}
+			>
+				<ClayTable borderedColumns={false} borderless>
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js:111]: can you remove the background color for this table?

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java [hunk 3]
```diff
@@ -1085,7 +1094,50 @@ private <T extends BaseModel<T>> Map<String, Object> _getWorkflowData(
...
 			WorkflowTask workflowTask = workflowTasks.get(0);
 
+			List<Integer> logTypes = new ArrayList<Integer>() {
+				{
+					add(WorkflowLog.TASK_ASSIGN);
+					add(WorkflowLog.TASK_COMPLETION);
+					add(WorkflowLog.TASK_UPDATE);
+					add(WorkflowLog.TRANSITION);
+				}
+			};
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java:1104]: Inline. You can use `Arrays.asList`

**Comment by brookedalton** [comment]
Hi @pei-jung, I tried putting the panel inside of the right table cell so the icon is on the far right but it looks like this

https://github.com/liferay-publications/liferay-portal/assets/46908843/f51ac5aa-926b-4d9c-b463-f1d7af2d40c1

**Comment by pei-jung** [comment]
@brookedalton having the panel inside the table looks weird. Let's stick with your previous change and try to adjust the spacing

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
https://github.com/liferay-publications/liferay-portal/assets/46908843/90855b9f-a353-469b-8c74-97d013dc66e3

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js [hunk 4]
```diff
@@ -3,94 +3,141 @@
...
+						collapsable
+						displayTitle={
+							<ClayPanel.Title>
+								<b> {Liferay.Language.get('activities')} </b>
+							</ClayPanel.Title>
+						}
+						displayType="primary"
+						showCollapseIcon={true}
+					>
+						<th style={{backgroundColor: 'white'}}>
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js:115]: Use `bg-white` class from [background color](https://clayui.com/docs/css/utilities/markup-color-utilities.html#css-background-color) helper.

**Review by pei-jung — CHANGES_REQUESTED** [review]
@brookedalton just one minor comment. Everything else looks great! 👍

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**